docs/pages/home/release-notes.md

```diff
@@ -5,6 +5,11 @@ toc_label: 'Version Number'
 sidebar: about_sidebar
 ---
 
+## 0.8.7
```

Comment:
This change is only to the docs, IMO we don't need to do a release for just that now. Good to fix it in master though.

Reply:
Hmmm, since the docs are included in the base simulato package it makes sense to me to have a release, especially to show good citizenship within the open source community by fixing security vulnerabilities promptly.  Perhaps in the future there should be consideration made to split them out so they can have independent release cycles.

Reply:
We don't **need** a release but either way is fine. If Mike wants to experience pushing a release then that could be good. Just follow the release docs: https://gannettdigital.github.io/simulato/publishing/. If you want to wait for this to got out in the next release just remove the changes here and put the changes under `## PendingVersion` in CHANGELOG.md here: https://github.com/GannettDigital/simulato/blob/master/CHANGELOG.md

Reply:
I agree with Mike that optimal open source etiquette would mean we should do a release.

Reply:
Latest commit is prepping for the release